sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/ChunkedDownloadUtils.java

```diff
@@ -117,10 +119,107 @@ public static <T> Flux<T> downloadChunk(Integer chunkNum, BlobDownloadAsyncRespo
 
         // Make the download call.
         return downloader.apply(chunkRange, requestConditions)
-            .subscribeOn(Schedulers.elastic())
+            .subscribeOn(Schedulers.boundedElastic())
             .flatMapMany(returnTransformer);
     }
 
+    /*
+    Download the first chunk. Construct a Mono which will emit the total count for calculating the number of chunks,
+    access conditions containing the etag to lock on, and the response from downloading the first chunk.
+     */
+    public static Mono<Tuple4<Long, BlobRequestConditions, StreamResponse, BlobRange>> streamFirstChunk(
```

Comment:
I'd lean towards turning this Tuple4 into an actual class that's easier to decipher